boards/microbit_v2/README.md

```diff
@@ -55,7 +132,7 @@ APP=../../../libtock-c/examples/c_hello/build/cortex-m4/cortex-m4.tbf
 KERNEL=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/debug/$(PLATFORM).elf
 KERNEL_WITH_APP=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/debug/$(PLATFORM)-app.elf
 
-.PHONY: program
+.PHONY: flash-app
```

Comment:
Here also, shouldn't we just refer users to the normal, tockloader based approach for installing apps?

Reply:
Done